docker-bake.hcl

```diff
@@ -57,8 +57,6 @@ target "artifact-all" {
   inherits = ["artifact"]
   platforms = [
     "linux/amd64",
-    "linux/arm/v6",
-    "linux/arm/v7",
```

Comment:
Yeah, no. Let's not do this. We can't remove these. There are binaries have been released in this arch as well as official images https://github.com/distribution/distribution-library-image/blob/a943e89c3efe06134cd9a4b439203c5341082cbc/Dockerfile shipping them.

Reply:
agree